vendor.conf

```diff
@@ -32,17 +32,17 @@ github.com/hashicorp/golang-lru                     v0.5.3
 github.com/imdario/mergo                            v0.3.7
 github.com/konsorten/go-windows-terminal-sequences  v1.0.3
 github.com/matttproud/golang_protobuf_extensions    v1.0.1
-github.com/Microsoft/go-winio                       v0.4.14
+github.com/Microsoft/go-winio                       fc70bd9a86b5562b3b5eb1040e803febee1e90a1
```

Comment:
Looks like there's 1 commit in this diff; https://github.com/Microsoft/go-winio/compare/v0.4.14...fc70bd9a86b5562b3b5eb1040e803febee1e90a1. Which package depends on that change, is that k8s or one of the other dependencies?

Should we ask @kevpar to tag a new release? (looks like there's a couple more changes in master; https://github.com/Microsoft/go-winio/compare/fc70bd9a86b5562b3b5eb1040e803febee1e90a1...master)

Reply:
I think this was a random change that ended up getting propagated through the dependency chains. tracked it down to:
https://github.com/google/cadvisor/commit/3b75b98f8f793de536d8c1344b82d9c471fcf193

Will just go back to v0.4.14